hydrolib/core/dflowfm/mdu/models.py

```diff
@@ -1263,6 +1400,75 @@ class Comments(INIBasedModel.Comments):
         richardsononoutput: Optional[str] = Field(
             "Write Richardson number, (1: yes, 0: no).", alias="richardsonOnOutput"
         )
+        wrimap_every_dt: Optional[str] = Field(
+            "Write output to map file every computational timestep, between start and stop time from MapInterval, (1: yes, 0: no).",
+            alias="wrimap_input_dt",
+        )
+        wrimap_input_roughness: Optional[str] = Field(
+            "Write chezy input roughness on flow links to map file, (1: yes, 0: no).",
+            alias="wrimap_input_roughness",
+        )
+        wrimap_flowarea_au: Optional[str] = Field(
+            "Write flow areas au to map file (1: yes, 0: no).",
+            alias="wrimap_flowarea_au",
+        )
+        wrihis_airdensity: Optional[str] = Field(
+            "Write air density to his file (1: yes, 0: no).", alias="wrihis_airdensity"
+        )
+        wrimap_flow_flux_q1_main: Optional[str] = Field(
+            "Write flow flux in main channel to map file (1: yes, 0: no).",
+            alias="wrimap_flow_flux_q1_main",
+        )
+        wrimap_windstress: Optional[str] = Field(
+            "Write wind stress to map file (1: yes, 0: no).", alias="wrimap_windstress"
+        )
+        wrishp_genstruc: Optional[str] = Field(
+            "Writing general structures to shape file (0=no, 1=yes).",
+            alias="wrishp_genstruc",
+        )
+        wrimap_qin: Optional[str] = Field(
+            "Write sum of all influxes to map file (1: yes, 0: no).", alias="wrimap_qin"
+        )
+        wrimap_dtcell: Optional[str] = Field(
+            "Write time step per cell based on CFL (1: yes, 0: no).",
+            alias="wrimap_dtcell",
+        )
+        wrimap_velocity_vectorq: Optional[str] = Field(
+            "Write cell-center velocity vectors (discharge-based) to map file (1: yes, 0: no).",
+            alias="wrimap_velocity_vectorq",
+        )
+        wrimap_bnd: Optional[str] = Field(
+            "Write boundary points to map file (1: yes, 0: no).", alias="wrimap_bnd"
+        )
+        wrishp_dambreak: Optional[str] = Field(
+            "Writing dambreaks to shape file (0=no, 1=yes).", alias="wrishp_dambreak"
+        )
+        wrimap_waterdepth_hu: Optional[str] = Field(
+            "Write water depths on u-points to map file (1: yes, 0: no).",
+            alias="wrimap_waterdepth_hu",
+        )
+        ncmapdataprecision: Optional[str] = Field(
```

Comment:
this is an int below, but this is talking about precision, should it be an int?

Reply:
I assume it is an `int`. The default value according to the very up-to-date manual is: `0` (an integer).

EDIT: I just checked the file Jelmer provided in the issue (the allow_extra file), there the description of this keyword more clearly indicates that it is indeed an integer with two valid values 0 and 1.

`Precision for NetCDF data in map files (0: double, 1: float)`